src/configurations/destinations/zoho_dev/db-config.json

```diff
@@ -38,6 +38,9 @@
     }
   },
   "options": {
-    "hidden": true
+    "hidden": {
+      "featureFlagName": "HIDE_ZOHO_DEV",
+      "featureFlagValue": false
```

Comment:
what does it mean:
it will be hidden when HIDE_ZOHO_DEV: false

Reply:
Yes

Reply:
It should look like: It reads as "_When the feature flag enable_zoho_dev is false then hide the definition_"

You still need to prefix the flag with `AMP_` when referencing it from the db-config.json.

the name of the actual flagsmith flag is just `enable-zoho-dev-destination`

```
  "options": {
    "hidden": {
      "featureFlagName": "AMP_enable-zoho-dev-destination",
      "featureFlagValue": false
    },
```

Reply:
@ItsSudip let's close this PR, I've updated another one I had open to fix an issue when connecting an Audience 🙇